webui/react/src/hooks/useFeature.ts

```diff
@@ -0,0 +1,25 @@
+import queryString from 'query-string';
+import { useLocation } from 'react-router-dom';
+
+import { useStore } from 'contexts/Store';
+
+interface FeatureHook {
+  isOnRbac: () => boolean
```

Comment:
Just to kind of get the ball rolling on using feature switches more broadly could we make this 
```
type ValidFeature = 'rbac' // Add new feature switches here using `|`
interface FeatureHook {
   isOn: (ValidFeature) => boolean
}
```
General behavior for `isOn(someString)` should be to check if the page was loaded with a query parameter of `?f_someString=on`. You can add a special case for rbac to also check the master config info.

Reply:
If we have `ValidFeature` as a "list" of features, should we have return value as a list, too?
Can we have `isOn` function just takes individual feature instead of a list?

Reply:
`ValidFeature` isn't a list, it's a type to prevent typos. For runtime purposes it's a string. So if you do `type ValidFeature = 'rbac' | 'triangle_mode' then `useFeature().isOn('rbac')` will compile but `useFeature().isOn('rbc')` will be a typescript error.

Reply:
I see! Thanks!